<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CafeManagerWindow</class>
 <widget class="QMainWindow" name="CafeManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cafe Manager Portal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="CafeResources.qrc">:/content/Copy of Untitled.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>All Bids</string>
     </attribute>
     <widget class="QPushButton" name="approveButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(97, 255, 137);
color:dimgray;</string>
      </property>
      <property name="text">
       <string>Approve ↓</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="bidTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rejectButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: red;</string>
      </property>
      <property name="text">
       <string>Reject X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Chef Bids</string>
     </attribute>
     <widget class="QTableWidget" name="chefTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>361</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="chefApproveButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(97, 255, 137);
color:dimgray;</string>
      </property>
      <property name="text">
       <string>Approve ↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chefRejectButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: red;</string>
      </property>
      <property name="text">
       <string>Reject X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cashier Bids</string>
     </attribute>
     <widget class="QTableWidget" name="cashierTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>371</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="cashierApproveButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(97, 255, 137);
color:dimgray;</string>
      </property>
      <property name="text">
       <string>Approve ↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cashierRejectButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: red;</string>
      </property>
      <property name="text">
       <string>Reject X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Waiter Bids</string>
     </attribute>
     <widget class="QTableWidget" name="waiterTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="waiterApproveButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(97, 255, 137);
color:dimgray;</string>
      </property>
      <property name="text">
       <string>Approve ↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="waiterRejectButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: red;</string>
      </property>
      <property name="text">
       <string>Reject X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Rejected Bids</string>
     </attribute>
     <widget class="QTableWidget" name="rejectedTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rejectedApprove">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(97, 255, 137);
color:dimgray;</string>
      </property>
      <property name="text">
       <string>Approve ↓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rejectedUnreject">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: dimgray;</string>
      </property>
      <property name="text">
       <string>Unreject</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>1021</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Workslot Status</string>
     </attribute>
     <widget class="QTableWidget" name="slotTable">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="unapproveButton">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
color: dimgray;</string>
      </property>
      <property name="text">
       <string>Unapprove ↑</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>210</y>
        <width>281</width>
        <height>28</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Chefs:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="chefText">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Cashiers:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cashierText">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Waiters:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="waiterText">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="staffTable">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>331</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Staff</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>253</width>
        <height>37</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Workslots for</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="showWorkslotsButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: dodgerblue;
color: dimgray;</string>
      </property>
      <property name="text">
       <string>( Show All Workslots )</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="CafeResources.qrc"/>
 </resources>
 <connections/>
</ui>
